### DISCUSSION TITLE
posts/quidditch-cyber/

### DISCUSSION THREAD

**Discussion by giscus** [azureeen/azureeen-blog#6] [Blog Azureeen]
# posts/quidditch-cyber/

Salut à tous les moldus ! Aujourd’hui, on va plonger dans le monde palpitant du Quidditch, mais pas pour discuter des derniers matchs à Poudlard. Non, on va voir comment ce sport fantastique peut nous éclairer sur les stratégies de cybersécurité. Alors, attrapez vos balais, on décolle !

https://azureeen.netlify.app/posts/quidditch-cyber/

**Comment by Aiglegg** [comment]
Très clair ! Bravo pour le lien entre cyber et Harry Potter. En tant que fan j'adore !
